scripts/run-clang-format.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+FORMATTER="clang-format"
+
+# install if the formatter is not found
+if ! command -v ${FORMATTER} &> /dev/null; then
+    echo "${FORMATTER} not found. Proceeding to install."  
+
+    # Install llvm (includes clang-format)
+    brew install llvm
+
+    export PATH="/opt/homebrew/opt/llvm/bin:$PATH"
+fi
+
+# 1) Paths to formart
+SOURCE_DIRS=(
+    "$(pwd)/integration-tests/basic-test-app/ios/basictestapp"
+    "$(pwd)/examples/react-native-test-suite"
+    "$(pwd)/packages"
+    "$(pwd)/packages/core/test-project"
+)
+
+# find command
+FIND_CMD="find"
+for dir in "${SOURCE_DIRS[@]}"; do
+    FIND_CMD+=" $dir"
+done
+
+# 2) Exclude directories. Add more directories if needed
+EXCLUDE_DIRS=(
+    "$(pwd)/node_modules"
+    "$(pwd)/build"
+    "$(pwd)/packages/core/scripts/__tests__"
```

Comment:
@jpmunz added this line here to avoid linting objective-c files under __tests__ since we are testing and comparing exact strings (which ends into failing cases)